assets/js/handlers.js

```diff
@@ -150,7 +150,10 @@
 
         // Focus search bar on '/'
         if (event.key == "/") {
-            document.getElementById('searchbox').focus();
+            let searchbox = document.getElementById('searchbox');
+            if (searchbox == document.activeElement) return;
```

Comment:
Should we ignore it if any text box is focused?

Reply:
While there is currently only one text box on every page (the search box) you're entirely right! This will avoid problems in the future!